pkg/utils/collect_errors_test.go

```diff
@@ -0,0 +1,43 @@
+package utils_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/utils"
+)
+
+func TestCollectSingleError(t *testing.T) {
+	errchan := make(chan error)
+	defer close(errchan)
+
+	go func() {
+		errchan <- returnError()
+	}()
+
+	err := utils.CollectErrors(errchan, 1*time.Second)
+	assert.Error(t, err)
+	assert.Equal(t, err.Error(), "error")
+}
+
+func TestCollecMultipleError(t *testing.T) {
+	errchan := make(chan error)
+	defer close(errchan)
+
+	go func() {
+		errchan <- returnError()
+		errchan <- returnError()
+		errchan <- returnError()
+	}()
+
+	err := utils.CollectErrors(errchan, 1*time.Second)
+	assert.Error(t, err)
+	assert.Equal(t, err.Error(), "error\nerror\nerror")
+}
+
+func returnError() error {
```

Comment:
remove this func and use inline `errors.New("error")`, possibly with different error messages.